media/baseset/orig_extra/fix_gui_icons.nfo

```diff
@@ -0,0 +1,12 @@
+// This file is part of OpenTTD.
+// OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+// OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+//
+   -1 * 0	 0C "Fix alignment of button icons."
+
+// Fix alignment of button icons.
+   -1 * 11	 0A 03 01 DC 02 01 E2 02 01 E6 02
+   -1 sprites/fix_gui_icons.png 8bpp   8  13  18  20  1  0 normal nocrop
```

Comment:
Isn't the `1` meaning that it should offset the sprite? Though that information is later going to be ignored when centering, right? So shouldn't it be just 0 like the next two sprites?
And shouldn't the sprite just be made 20x20 pixels? Although I am assuming that all the other buttons are also 20x20 pixels.

Reply:
I was trying to visually match the old alignment, and just using 20 still had a pixel difference (from whatever reason).

Reply:
Ideally it should be 20x20 with 0x0 offset. It is likely to not exactly match what it used to be before as it was incorrectly aligned then too. And the button itself is bigger due to padding being scaled.